### DISCUSSION TITLE
Model for a single closed population

### DISCUSSION THREAD

**Discussion by anyperez** [RangeShifter/RangeshiftR-tutorials#6] [Q&A] [answered]
Hi all,

I'm trying to do a stage-structured only-female model for a single closed population on the RangeShiftR package. As there is no dispersal, I'm using only one cell on the landscape module, and on the demography module, these are the vital rates for the four stages that I want to introduce: SurvR pup = 0.5, juv = 0.9, adult = 0.97, sen = 0.71; FertR adult = 0.35, sen = 0.05; Develop(gamma) pup-juv = 1, juv-adult = 0.2, adult-sen = 0.05. The initial pop size would be 84,000 females. Given this parameterization, my script is the following:

sim <- Simulation(Simulation = 1, Years = 100, Replicates = 1)
land <- ArtificialLandscape(propSuit = 1, Resolution = 100,  dimX = 1, dimY = 1)
stg <- StageStructure(Stages = 4, TransMatrix = matrix(c(0,.5,0,0,0,.72,.18,0,.35,0,.92,.05,.05,0,0,.71),nrow = 4), MaxAge = 50, MinAge = c(0,0,6,26))
demo <- Demography(StageStruct = stg)
init <- Initialise(InitDens = 2, IndsHaCell = 84000, PropStages = c(0,.34,.58,.08))
s <- RSsim(simul = sim, land = land, demog = demo, init = init)
validateRSparams(s)
RunRS(s, dirpath)

However, the mean growth rate (lambda) that I get for this model (1.4%) is not what I expected for this population (7-10%), and I'm struggling to get that expected growth rate increasing the vital rates, even up to unrealistic values. **I'd like to know if I'm doing something wrong based on the problem that I want to tackle (maybe I'm missing important details on how the code works), or if you have any suggestions to improve the model**.

Many thanks

**Comment by damariszurell** [comment, marked as answer]
Hi there, is your expectation of 7-10% growth based on empirical observations or based on simple matrix simulations from your transition matrix? You could also "force" faster population growth by lowering the minimal age for the different stages.
